src/apps/loan-list/modal/renew-button.tsx

```diff
@@ -48,13 +48,12 @@ const RenewButton: FC<RenewButtonProps> = ({
   );
 
   return (
-    <div className="modal-details__buttons">
+    <div className={`modal-details__buttons ${classNames}`}>
```

Comment:
Consider using `clsx` for constructing `className` values. This is quite a simple case so here it is OK to not do so.